src/prime_rl/orchestrator/orchestrator.py

```diff
@@ -273,7 +276,6 @@ async def orchestrate(config: OrchestratorConfig):
 
         # Log progress metrics to monitor
         progress_metrics = {
-            "progress/orchestrator/total_tokens": progress.total_tokens,
             "progress/orchestrator/total_samples": progress.total_samples,
             "progress/orchestrator/epoch": progress.epoch,
```

Comment:
lets delete the orchestrator from these then tho
```suggestion
            "progress/total_samples": progress.total_samples,
            "progress/epoch": progress.epoch,
```